lib/field-locale.ts

```diff
@@ -12,9 +12,9 @@ export default class FieldLocale implements FieldAPI {
   items?: Items
   private _value: any
 
-  private _valueSignal: MemoizedSignal
-  private _isDisabledSignal: MemoizedSignal
-  private _schemaErrorsChangedSignal: MemoizedSignal
+  private _valueSignal: MemoizedSignal<[any]>
```

Comment:
same here.

Reply:
`any` here is fine. That's what we use for field values within the App SDK.